### DISCUSSION TITLE
Just landed from Google; roadmap is old; project status is...?

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [spacedriveapp/spacedrive#687] [General]
Hi everyone. I'm new here. 👋

This project looks amazing. I believe it fills a massive hole in the general personal computing landscape. I stumbled across it half an hour ago via a Google search for the phrase "cross platform file tagging". I was hyped right away. 🤩

My hype only took a few minutes to _partly_ fizzle out however, once I noticed that the roadmap considers "present day" to be roughly Q3 2022. Unless Spacedrive is so intensely "from the future" that it runs on a whole other space-time continuum, the first full release of this project—Q2 2023 according to the roadmap—could land as soon as tomorrow.

It's encouraging to see you guys have got $2M in funding, but I'd bet _$20_ that Spacedrive isn't launching tomorrow. The last (and only) blog post was 8 months ago. I can see from this repo that the project is quite active, but if I had gone off of the website alone, I'd have considered it well on the way to archive-land.

**I didn't come here to roast this project.** I love the idea, I believe in the vision, I believe that this project is one among many that could help change computing as we know it thanks to Rust 🦀, and I can't wait to be among the first to give it a try. But if you want people (such as myself) to really get behind you, please consider my plea for some communication. I see so many projects these days that suddenly appear with big promises, and then disappear just as fast. Personally, what I look for in these sorts of projects is regular roadmap updates or blog posts, perhaps once every 3 months (once a quarter) at minimum.

Just give us 2 paragraphs about what you guys worked on this month, with a cool screenshot or something. At best, you'll get a whole bunch of buzz around the project for free. At worst, you're building trust with people who are watching the project.

All the best!

**Comment by jamiepine** [comment]
Hey! Really nice to hear the positive words about the project and I understand the frustration with the lack of communication on our end. I am currently working on an update blog post detailing our progress, as well as a roadmap update. Until now we have been heads down internally, as much as I feel bad for not keeping the website up to date, at the very least being open source it is evident our team is hard at work. Will make sure to keep progress reports frequent going forward.
